pandas/io/parsers.py

```diff
@@ -1474,6 +1474,13 @@ def _set(x):
                 else:
                     _set(val)
 
+        elif self.parse_dates:
+            if isinstance(self.index_col, list):
```

Comment:
``is_list_like``? (I don't know if its coerced to a list before this), same below